tests/testthat/test-get.R

```diff
@@ -365,3 +374,86 @@ test_that("get_content only requests vanity URLs for Connect 2024.06.0 and up",
     )
   })
 })
+
+test_that("get_usage() returns usage data in the expected shape", {
```

Comment:
Could we add a test where we pass in a character that is a formatted timestamp? (if we are intending to support that, of course)